docs/content/configuration/index.md

```diff
@@ -1023,7 +1023,7 @@ Worker select strategies control how Druid assigns tasks to middleManagers.
 
 ###### Equal Distribution
 
-Tasks are assigned to the middleManager with the most available capacity at the time the task begins running. This is
+Tasks are assigned to the middleManager with the most spare capacity at the time the task begins running. This is
```

Comment:
I don't know.

Reply:
Let's do `Tasks are assigned to the middleManager with the most free slots at the time`

Reply:
Also, let's write `Middle Manager`

Reply:
Applied the first suggestion. I didn't replace the term "middleManager" (one word) with "Middle Manager" because the first version is used throughout the document in many places. I'm also not sure that the separate writing is better for searching in the docs. If you feel that the replacement would be an improvement, please open an issue.